File: programmers/default_minMax.py
# rank:	    2
# title:	최댓값과 최솟값
# link: 	https://programmers.co.kr/learn/courses/30/lessons/12939

def solution(s):
    arr = [int(num) for num in s.split(" ")]
    return "{} {}".format(min(arr), max(arr))

File: programmers/etc/javascript/default/default_minMax.js
/*
	rank:	2
	title:	최댓값과 최솟값
	link: 	https://programmers.co.kr/learn/courses/30/lessons/12939
*/

function solution(s) {
    let arr = s.split(" ");
    return `${Math.min(...arr)} ${Math.max(...arr)}`;
}
